alembic/versions/9359297269a8_convert_ribbon_to_a_multichoice_column.py

```diff
@@ -0,0 +1,64 @@
+"""Convert ribbon to a multichoice column.
+
+Revision ID: 9359297269a8
+Revises: 167243c0e86c
+Create Date: 2017-07-21 09:18:38.444238
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '9359297269a8'
+down_revision = '167243c0e86c'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table
+
+
+try:
+    is_sqlite = op.get_context().dialect.name == 'sqlite'
+except:
+    is_sqlite = False
+
+if is_sqlite:
+    op.get_context().connection.execute('PRAGMA foreign_keys=ON;')
+    utcnow_server_default = "(datetime('now', 'utc'))"
+else:
+    utcnow_server_default = "timezone('utc', current_timestamp)"
+
+# We need this table in order to upgrade/downgrade the ribbon column
+attendee = table(
+    'attendee',
+    sa.Column('ribbon', sa.Unicode())
+)
+
+def upgrade():
+    op.alter_column('attendee', 'ribbon', type_=sa.Unicode(), server_default = '', nullable = False)
+
+    op.execute(
+        attendee
+            .update()
+            .where(attendee.c.ribbon == '154973361')
+            .values({'ribbon': ''})
+    )
+
+
+def downgrade():
+    op.execute(
+        attendee
+            .update()
+            .where(attendee.c.ribbon == '' or attendee.c.ribbon.contains(','))
+            .values({'ribbon': 154973361})
+    )
+
+    op.alter_column('attendee', 'ribbon', server_default=None, nullable=True)
```

Comment:
Casting from a unicode to a integer is an interesting challenge! As far as I can tell, we have to:
1. Remove the server default, which just straight-up cannot be cast to an integer
2. Update the column to type integer, using raw SQL because postgres requires extra syntax that alembic doesn't support
3. Reinstate the server default

This is enough to get the downgrade to run, which I needed to do on my dev box (and is a valid use-case during development in any case).